src/corehost/cli/deps_resolver.cpp

```diff
@@ -164,6 +164,18 @@ void deps_resolver_t::setup_shared_store_probes(
     }
 }
 
+pal::string_t deps_resolver_t::get_probe_directories()
+{
+    pal::string_t directories;
+    for (const auto& pc : m_probes)
```

Comment:
I agree some locations don't make sense, but i would rather not put more logic in the host. This will also function in addition to what @eerhardt mentioned, a way to expose what paths host is actually trying to look up assemblies.